fe/README.md

```diff
@@ -1,16 +1,30 @@
-# smart_turner
+# SmartTurner - Running the Flutter App
 
-A new Flutter project.
 
-## Getting Started
+## Install backend dependencies
+```
+cd be
+npm install --immutable
+```
 
-This project is a starting point for a Flutter application.
+## Run the server
+Inside `/be`, create a `.env` file with the contents `NODE_PORT=3000`. Now we can run the server at port 3000:
```

Comment:
You can write `NODE_PORT=<your port here>`

Reply:
Same with `PYTHON_PORT` in the future

Reply:
Sounds good